src/i18n.js

```diff
@@ -57,10 +57,16 @@ for (let lang in languageList) {
 
 const rtlLangs = [ "fa", "ar-SY", "ur" ];
 
-export const currentLocale = () => localStorage.locale
-    || languageList[navigator.language] && navigator.language
-    || languageList[navigator.language.substring(0, 2)] && navigator.language.substring(0, 2)
-    || "en";
+/**
+ * Find the best matching locale to display
+ * If no locale can be matched, the default is "en"
+ * @returns {string} the locale that should be displayed
+ */
+export function currentLocale() {
+    const potentialLocales = [ localStorage.locale, navigator.language, navigator.language.substring(0, 2), ...navigator.languages ];
+    const availableLocales = potentialLocales.filter(l => languageList[l]);
```

Comment:
This will always skip English because `en` (or its variants like `en-US`) is present in `languageList`. Users with browser configured to send multiple supported locales will always have interface rendered in first non-English language.

Reply:
It's also important to note that `en` is not guaranteed to be present in `navigator.languages` and instead one of its variant (`en-US`, `en-GB`) will be present as only choice. This should be accounted for!

Reply:
Well having browsers have the more specificlocale before the less specific loclale in my testing.
Is this an actual issue that you have experienced or just a hypothetical? Could you report what `navigator.languages` is on your device?

![Screenshot_20240419_025002_Chrome.jpg](https://github.com/louislam/uptime-kuma/assets/26258709/5d8e1c22-7bb6-4955-bf03-7d99037355aa)